Topics/Teamwork.md

```diff
@@ -1,5 +1,25 @@
 ## Resources for Teamwork
 
+## Communication
+
+
```

Comment:
You can also include the Challenges to effective communication:
like Language and cultural differences, Technology barriers, Time zone differences and/or Personal biases and emotions